🐛 Debug fails when admin token is less than 3 characters long

**Describe the bug**
Debug fails when admin token is not present in the data directory

**To Reproduce**
Steps to reproduce the behavior:
1. Replace data_dir/algod.admin.token with a string less than 3 characters long, e.g. `a`
2. Run `./nodekit debug`
3. Panics:

```
panic: runtime error: slice bounds out of range [:3] with length 0

goroutine 1 [running]:
github.com/algorandfoundation/nodekit/cmd.init.func1(0xc00025f808, {0xd5a193?, 0x4?, 0xd5a053?})
	/home/bit/code/nodekit/cmd/debug.go:94 +0x6ee
...
```

**Expected behavior**
No panic, debug output displayed

--- a/cmd/debug.go
+++ b/cmd/debug.go
@@ -90,7 +90,7 @@ var debugCmd = cmdutils.WithAlgodFlags(&cobra.Command{
 		folderDebug, err := utils.ToDataFolderConfig(dataDir)
 		if err != nil {
 			folderDebug.Token = fmt.Sprint(err)
-		} else {
+		} else if len(folderDebug.Token) > 3 {
 			folderDebug.Token = folderDebug.Token[:3] + "..."
 		}
 

